cmd/meroxa/turbine_cli/utils.go

```diff
@@ -265,13 +265,13 @@ func GoInit(ctx context.Context, l log.Logger, appPath string, skipInit, vendor
 			l.Error(ctx, string(output))
 			return err
 		}
-		cmd = exec.Command("go", "get", "github.com/meroxa/turbine")
+		cmd = exec.Command("go", "get", "github.com/meroxa/turbine-go")
```

Comment:
was this repo renamed before the last rerun of this failed job?

Reply:
master on turbine-go had an issue in go.mod.  just needed to merge the open PR and this started working much better.